crawl-ref/docs/develop/arena.txt

```diff
@@ -149,6 +149,6 @@ example:
     crawl -arena "arena:smoke random v random"
 
 will cause the monsters to fight in an arena which is filled with smoke. The
-available alternate terrains are in source/dat/arena.des. If an arena you
+available alternative terrains are in source/dat/arena.des. If an arena you
```

Comment:
We don't enforce the aus English thing in developer documentation. Also, fwiw this (really any change that tries to avoid adjectival "alternate") reads really awkwardly in us English. I would reword this to avoid the word altogether, in this case probably simply dropping the word: "The available alternative terrains are in source/dat/arena.des."

Reply:
There were a few places where I felt the word didn't add anything to the sentence it was in, but I didn't want to make changes I couldn't objectively justify. I've dropped this now and corrected the file location on that line.

I do wonder whether this could be merged with `docs/arena.txt`, though.